app/components/standup-deck.js

```diff
@@ -29,6 +29,7 @@ export default Component.extend(EKMixin, {
 
   actions: {
     goToNextSlide() {
+      const totalLength = this.startups.length + this.otherIncubators.length;
```

Comment:
À mettre directement avant les actions plutôt que de le répéter à chaque fois

Reply:
(et il faut rajouter au moins un "-1" sur les deux, non ? en fait si t'écris `(this.startups.length - 1) + (this.otherIncubators.length - 1) + 1`, ça sera plus clair, donc autant écrire ça)

Reply:
> À mettre directement avant les actions plutôt que de le répéter à chaque fois

Je ne sais pas comment faire.

> et il faut rajouter au moins un "-1" sur les deux, non 

Non, si je suis rendu à l'index 2 des startups sur 4 et qu'il y a 2 incubateurs et 1 meta ça fait un total de `2 / 7` ce qui est juste. Au début je suis à `0 / 7` et à la fin à `7 / 7`. Quand je teste ça marche bien d'ailleurs. Donc à la limite ça pourrait être : 

```
this.startups.length + this.otherIncubators.length + 1 /* meta */ -1;
```
Mais ça me semble plus confus

Reply:
En effet, je me suis emmelé les pinceaux :) cela dit, je ne te suis pas sur le total. Pour moi dans ton exemple le total sera de 6, non ? (4 + 2)

Reply:
haha complexe : en effet le total est 6 si on compte de 0 à 6. Tu as donc raison, les intentions ne sont pas claires :)